src/OnlySpans.PolyLeads.Api.Schema/Inputs/Users/Sign/SignInKey/Email.cs

```diff
@@ -0,0 +1,10 @@
+﻿using HotChocolate;
+
+namespace OnlySpans.PolyLeads.Api.Schema.Inputs.Users.Sign.SignInKey;
+
+[GraphQLDescription("Почта пользователя - необходима для входа в аккаунт")]
```

Comment:
Так подробно не расписывай
```suggestion
[GraphQLDescription("Почта")]
```